monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -111,10 +112,22 @@ def get_tech_base_data(cls):
         data.update({'status': status,
                      'title': title,
                      'message': cls.get_message_by_status(status)})
+        data.update(cls.get_mitigation_by_status(status))
         return data
 
     @classmethod
     def get_base_data_by_status(cls, status):
         data = cls.get_message_and_status(status)
         data.update({'title': cls.technique_title()})
+        data.update(cls.get_mitigation_by_status(status))
         return data
+
+    @classmethod
+    def get_mitigation_by_status(cls, status: ScanStatus) -> dict:
+        if status == ScanStatus.USED.value:
+            mitigation_document = AttackMitigation.get_mitigation_by_technique_id(str(cls.tech_id))
```

Comment:
This might throw, I think you want to catch it here and return a const "mitigation not found" JSON

Reply:
this will throw if developer failed and somehow left a typo in tech_id. In that case it should not go by unnoticed